### DISCUSSION TITLE
Pending job stuck?

### DISCUSSION THREAD

**Discussion by bmtcril** [renovatebot/renovate#23281] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Job 01893d22-4e8d-79e1-91f8-1deafef7eef9 is stuck in pending, and we would have expected it to complete ~7.5 hrs ago. Other jobs have run on this repo, but that would be the first one we'd expect to generate a PR so we're not sure if there's a configuration issue or if there's just a lag on the job runner. We have other repos running successfully.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Please include a screenshot of the job list

**Comment by bmtcril** [comment]
@rarkins  here you go:

<img width="1206" alt="Screenshot 2023-07-10 at 12 28 11 PM" src="https://github.com/renovatebot/renovate/assets/112640379/bdd4ef20-a057-4023-88c0-25ff79e79a13">

**Reply by rarkins** [reply]
The job is not stuck. Pending does not mean "in progress",
It means waiting to start. Daily jobs are usually executed within a day. Merge a PR to get the repo prioritized to every ~3 hours

**Comment by bmtcril** [comment]
Just so I understand, this repo is configured for `schedule:weekly`, which extends `schedule:earlyMondays`, which extends ` "before 4am on Monday"` according to the [docs](https://docs.renovatebot.com/presets-schedule/#scheduleweekly), but in actuality it can be any time on Monday?

**Reply by rarkins** [reply, marked as answer]
Yes, repos with no merged PRs get a scheduled job enqueued once per day. You can request a job manually using the dependency dashboard issue
